<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IVCalculator</class>
 <widget class="QWidget" name="IVCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IV Calculator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxSettings">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelSpecies">
        <property name="text">
         <string>Pok√©mon</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelNature">
        <property name="text">
         <string>Nature</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelGeneraiton">
        <property name="text">
         <string>Generation</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBoxPokemon"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxNature"/>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="comboBoxGeneration">
        <item>
         <property name="text">
          <string>Gen 3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gen 4</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelCharacteristic">
        <property name="text">
         <string>Characteristic</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelHiddenPower">
        <property name="text">
         <string>Hidden Power</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QComboBox" name="comboBoxCharacteristic"/>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBoxHiddenPower">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>219</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pushButtonFindIVs">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Find IVs</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="labelLevel">
          <property name="text">
           <string>Level</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelHP">
          <property name="text">
           <string>HP</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="labelAtk">
          <property name="text">
           <string>Atk</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="labelDef">
          <property name="text">
           <string>Def</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="labelSpA">
          <property name="text">
           <string>SpA</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLabel" name="labelSpD">
          <property name="text">
           <string>SpD</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QLabel" name="labelSpe">
          <property name="text">
           <string>Spe</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSpinBox" name="spinBoxLevel">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBoxHP">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>800</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="spinBoxAtk">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>800</number>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QSpinBox" name="spinBoxDef">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>800</number>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QSpinBox" name="spinBoxSpA">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>800</number>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QSpinBox" name="spinBoxSpD">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>800</number>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QSpinBox" name="spinBoxSpe">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>800</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxResults">
     <property name="title">
      <string>Results</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="labelHPIV">
        <property name="text">
         <string>HP</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelHPIVValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelAtkIV">
        <property name="text">
         <string>Atk</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelAtkIVValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelDefIV">
        <property name="text">
         <string>Def</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelDefIVValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelSpAIV">
        <property name="text">
         <string>SpA</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="labelSpAIVValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelSpDIV">
        <property name="text">
         <string>SpD</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="labelSpDIVValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelSpeIV">
        <property name="text">
         <string>Spe</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="labelSpeIVValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBoxInformation">
     <property name="title">
      <string>Information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="labelBaseHP">
        <property name="text">
         <string>Base HP</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelBaseHPValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelBaseAtk">
        <property name="text">
         <string>Base Atk</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelBaseAtkValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelBaseDef">
        <property name="text">
         <string>Base Def</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelBaseDefValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelBaseSpA">
        <property name="text">
         <string>Base SpA</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="labelBaseSpAValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelBaseSpD">
        <property name="text">
         <string>Base SpD</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="labelBaseSpDValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelBaseSpe">
        <property name="text">
         <string>Base Spe</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="labelBaseSpeValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
